### PR TITLE
config.json: Add topics & difficulty for poker

### PR DESCRIPTION
**Pull request by tleen**
See #345 & #731

### CODE REVIEW

**Comment by tleen** [comment]
"Domain-specific languages" may be a bit of a stretch. I figure using the combination of card values as a string with a relative value is a small but very domain-specific language.

File: config.json [hunk 1]
```diff
@@ -872,9 +872,13 @@
       "slug": "poker",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
```
Comment (leenipper) [inline config.json]: Difficulty seems about right.  Might go with 6.

File: config.json [hunk 2]
```diff
@@ -872,9 +872,13 @@
...
       "topics": [
-
+        "Domain-specific languages",
+        "Equality",
+        "Games",
+        "Parsing",
+        "Strings"
```
Comment (leenipper) [inline config.json]: What do you think about "Pattern matching", and "Sequences" as candidate topics ?
A poker "straight" is like a fixed size sequence. A full-house is like a pattern 3 of something and 2 of something else.

I agree that Domain-specific languages is stretching it a bit.  There are objects/nouns (the cards), but no actions/verbs for "the language".

Comment (tleen) [inline config.json]: Yep I think those two are a better descriptive replacement for DSL.

**Pull request merged**